src/hotspot/share/gc/shenandoah/shenandoahMmuTracker.hpp

```diff
@@ -50,16 +50,32 @@ class ShenandoahMmuTask;
  * MMU.
  */
 class ShenandoahMmuTracker {
+private:
+  // For reporting utilization during most recent GC cycle
```

Comment:
May be elaborate a bit (please correct as needed):
```
// These hold the values of the most recent snapshots of cumulative
// observations, and are used to compute deltas for specific 
// recording/reporting epochs at each subsequent snapshot.
```

Reply:
Thanks.  I've enhanced these comments in the expand-old-on-demand branch.